### PR TITLE
fix: fixed navbar in all pages

### PR DESCRIPTION
**Pull request by zabihshaik**
Fixed Navbar in all pages of ecom.

Navbar looks like:

Fixed Navbar

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
Looks good merge.

File: index.html [hunk 1]
```diff
@@ -38,11 +38,11 @@ <h3>Cric Zone</h3>
                     <i class="fa fa-search"></i>
```
Comment (sah-shah7) [inline index.html]: Nav bar looks ok, just icon alignment is not proper

**Review by anandkumarcoder — APPROVED** [review]
looks good to me

**Comment by zabihshaik** [comment]
fixed minor changes as per PR Review

**Pull request merged**